tokio-util/tests/length_delimited.rs

```diff
@@ -423,9 +426,15 @@ fn write_single_frame_length_adjusted() {
     pin_mut!(io);
 
     task::spawn(()).enter(|cx, _| {
-        assert_ready_ok!(io.as_mut().poll_ready(cx));
+        assert_ready_ok!(<MockFramedWrite as futures::Sink<Bytes>>::poll_ready(
+            io.as_mut(),
+            cx
+        ));
```

Comment:
As discussed in the original issue, this change could break the existing code like this, so I'm not sure we can do this change right now.